### DISCUSSION TITLE
docs/v1.10.x/guide/subnet/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2863] [Announcements]
# docs/v1.10.x/guide/subnet/

https://kubeovn.github.io/docs/v1.10.x/guide/subnet/

**Comment by linkinghack** [comment]
文档中对于Overlay模式分布式网关的描述：“当 natOutgoing 为 true 时，Pod 访问外部网络将会使用当前所在宿主机的 IP。” 不正确。 经过实验，分布式网关模式下出集群的流量被转发到了与出访行为Pod不是同一个节点的主机上，看起来是“随机选择一个宿主机作为NAT出口网关”

**Reply by zbb88888** [reply]
可以提个issue 一起看看，分布式场景下，应该会基于port group的策略路由到本地node的ovn0
